src/ui/components/Sidebar/Sidebar.jsx

```diff
@@ -39,7 +39,7 @@ export default function Sidebar(props) {
           <NavLink
             to={prop.layout + prop.path}
             className={activePro + classes.item}
-            activeClassName='active'
+            activeclassname='active'
```

Comment:
I think the fix here is actually to attach the `exact` property and keep the usage of `activeClassName` the same?

Ref: https://stackoverflow.com/questions/40218050/react-router-link-activeclassname-not-working

Reply:
> I think the fix here is actually to attach the `exact` property and keep the usage of `activeClassName` the same?
> 
> Ref: https://stackoverflow.com/questions/40218050/react-router-link-activeclassname-not-working

This post is quite old; my understanding is that `activeClassName` was deprecated, see https://dev.to/gabrlcj/react-router-v6-some-of-the-new-changes-181m#navlink-activeclassname-prop-does-not-exists-anymore

Reply:
@maoo - from the article, it looks like we want the following:

```js
className={(navData) => navData.isActive ? "active" : "" }
```

Reply:
> @maoo - from the article, it looks like we want the following:
> 
> ```js
> className={(navData) => navData.isActive ? "active" : "" }
> ```

Just tried, doesn't work...

Reply:
@maoo - can we redact this change for now as this doesn't fix the issue and we can look to put into another issue and merge your other changes?